test/models/patient_test.rb

```diff
@@ -3148,3 +3159,58 @@ def verify_patient_status(patient, status)
   end
 end
 # rubocop:enable Metrics/ClassLength
+
+# Inheriting PatientTest and overriding setup
+# allows us to run the same exact tests but change the Timecop time that
+# the tests are running at
+class PatientTestTwo < PatientTest
```

Comment:
Can you name these `PatientTestFromStandardTimeToDaylightSavingsTime` and from daylight to standard? Same with notification eligibility